compat-data-schema.md

```diff
@@ -115,7 +115,7 @@ No browser identifier is mandatory.
 #### The `support_statement` object
 The `support_statement` object describes the support provided by a single browser type for the given subfeature.
 It is an array of `simple_support_statement` objects, but if there
-is only one of them, the array can be omitted.
+is only one of them, the array should be omitted.
```

Comment:
must instead of should? The schema forbid it and will rise an error (My rule of thumb: must should lead to an error, should to a warning, can to nothing)

Reply:
Sounds good to me. Updated.